Tools/Cli-LoRa-Device-Provisioning/README.md

```diff
@@ -1,248 +1,3 @@
-```
-
-The update verb supports the following parameters:
-
-|parameter|required|description|
-|-|-|-|
-|--deveui|yes|DevEUI / Device Id: A 16 bit hex string.|
-|--appskey|no|AppSKey (Only for ABP devices): A 16 bit hex string.|
-|--nwkskey|no|NwkSKey (Only for ABP devices): A 16 bit hex string.|
-|--devaddr|no|DevAddr (Only for ABP devices): A 4 bit hex string.|
-|--netid|no|Network ID (Only for ABP devices): A 3 bit hex string. Will default to 000001 or NetId set in settings file if left blank.|
-|--abprelaxmode|no|ABPRelaxMode (ABP relaxed framecounter, only for ABP devices): True or false. |
-|--appeui|no|AppEUI (only for OTAA devices): A 16 bit hex string.|
-|--appkey|no|AppKey (only for OTAA devices): A 16 bit hex string.|
-|--gatewayid|no|GatewayID: A hostname. |
-|--decoder|no|SensorDecoder: The name of an integrated decoder function or the URI to a decoder in a custom decoder module in the format: <http://modulename/api/decodername>. |
-|--classtype|no|ClassType: "A" (default) or "C". |
-|--downlinkenabled|no|DownlinkEnabled: True or false. |
-|--preferredwindow|no|PreferredWindow (Preferred receive window): 1 or 2. |
-|--deduplication|no|Deduplication: None (default), Drop or Mark. |
-|--rx2datarate|no|Rx2DataRate (Receive window 2 data rate, currently only supported for OTAA devices): Any of the allowed data rates. EU: SF12BW125, SF11BW125, SF10BW125, SF8BW125, SF7BW125, SF7BW250 or 50. US: SF10BW125, SF9BW125, SF8BW125, SF7BW125, SF8BW500, SF12BW500, SF11BW500, SF10BW500, SF9BW500, SF8BW500, SF8BW500.|
-|--rx1droffset|no|Rx1DrOffset (Receive window 1 data rate offset, currently only supported for OTAA devices): 0 through 15.|
-|--rxdelay|no|RxDelay (Delay in seconds for sending downstream messages, currently only supported for OTAA devices): 0 through 15.|
-|--keepalivetimeout|no|KeepAliveTimeout (KeepAliveTimeout (Timeout in seconds before device client connection is closed): 0 or 60 and above.|
-|--supports32bitfcnt|no|Supports32BitFCnt (Support for 32bit frame counter): True or false. |
-|--fcntupstart|no|FCntUpStart (Frame counter up start value): 0 through 4294967295.|
-|--fcntdownstart|no|FCntDownStart (Frame counter down start value): 0 through 4294967295.|
-|--fcntresetcounter|no|FCntResetCounter (Frame counter reset counter value): 0 through 4294967295.|
-|--help|no|Display this help screen.|
-|--version|no|Display version information.|
-
-## remove
-
-Remove an existing device from IoT Hub by it's DevEUI / Device Id.
-
-Example:
-
-```powershell
-dotnet run -- remove --deveui 33CCC86800430010
-```
-
-The query verb supports the following parameters:
-
-|parameter|required|description|
-|-|-|-|
-|--deveui|yes|DevEUI / Device Id.|
-|--help|no|Display this help screen.|
-|--version|no|Display version information.|
+> !This file has been moved to a different branch: <https://github.com/Azure/iotedge-lorawan-starterkit/blob/docs/main/docs/tools/device-provisioning.md>
```

Comment:
Saying branch here is confusing since it's just a plain URL so suggest to change to simply:

```suggestion
This file has been moved to: <https://github.com/Azure/iotedge-lorawan-starterkit/blob/docs/main/docs/tools/device-provisioning.md>
```